src/Symfony/Component/Validator/Constraints/Ip.php

```diff
@@ -82,5 +84,9 @@ public function __construct($options = null)
         if (!\in_array($this->version, self::$versions)) {
             throw new ConstraintDefinitionException(sprintf('The option "version" must be one of "%s"', implode('", "', self::$versions)));
         }
+
+        if (null !== $this->normalizer && !\is_callable($options['normalizer'])) {
+            throw new \InvalidArgumentException(sprintf('The "normalizer" option must be a valid callable ("%s" given).', \is_object($this->normalizer) ? \get_class($this->normalizer) : \gettype($this->normalizer)));
```

Comment:
Don't we also have an `InvalidArgumentException` in the `Exception` namespace of the Validator component?